.circleci/config.yml

```diff
@@ -191,24 +179,3 @@ jobs:
           name: Create Github release and upload artifacts
           command: ghr -t $GITHUB_TOKEN -u $CIRCLE_PROJECT_USERNAME -r $CIRCLE_PROJECT_REPONAME --replace $CIRCLE_TAG bin/
 
-  publish-dev:
-    docker:
-      - image: circleci/golang:1.14
-    steps:
-      - attach_to_workspace
-      - setup_remote_docker
-      - run:
-          name: Build image
-          command: |
-            make docker-otelcol
-            docker tag otelcol:latest otel/opentelemetry-collector-dev:${CIRCLE_SHA1}
-            docker tag otelcol:latest otel/opentelemetry-collector-dev:latest
-      - run:
-          name: Login to Docker Hub
-          command: docker login -u $DOCKER_HUB_USERNAME -p $DOCKER_HUB_PASSWORD
-      - run:
-          name: Push image
-          command: |
-            docker push otel/opentelemetry-collector-dev:${CIRCLE_SHA1}
-            docker push otel/opentelemetry-collector-dev:latest
-
```

Comment:
You can keep this here.

Reply:
CI is too slow. It'll take another 20 mins :) 

I'll add it back along with the fix to enable publishing dev image again.